src/plugins/vis_type_timeseries/public/application/components/panel_config/timeseries.tsx

```diff
@@ -405,6 +405,7 @@ export class TimeseriesPanelConfig extends Component<
           <EuiTab
             isSelected={selectedTab === PANEL_CONFIG_TABS.DATA}
             onClick={() => this.switchTab(PANEL_CONFIG_TABS.DATA)}
+            data-test-subj="timeSeriesEditorDataBtn"
```

Comment:
nit: looks like you have never used that `data-test-subj`

Reply:
it is used in some tests to click on that tab like `PageObjects.visualBuilder.clickDataTab('timeSeries');`

Reply:
thank you